File: leetcode/question001/main.js
var twoSum = function(nums, target) {
    for(var i=0;i<nums.length;i++){
        var number1 = nums[i];
        for(var j=i+1;j<nums.length;j++){
            var number2 = nums[j];
            if(number1+number2==target){
                return [i,j];
            }
        }
    }
};

File: leetcode/question001/main.py
class Solution(object):
    def twoSum(self, nums, target):
        size = len(nums)
        for i in range(size):
            for j in range(size):
                if(nums[i]+nums[j]==target and i!=j):
                    return [i,j]

def main():
    solution = Solution()
    print(solution.twoSum([3,2,4], 6))

if __name__ == '__main__':
    main()

